modules/insights-operator-new-pull-secret-enable.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/disabling-insights-operator.adoc
+
+[id="insights-operator-new-pull-secret-enable_{context}"]
+= Modifying your global cluster pull secret to enable remote health reporting
+
+You can modify your existing global cluster pull secret to enable remote health reporting. If you have previously disabled remote health monitoring, you must first download a new `cloud.openshift.com` access token from Red Hat OpenShift Cluster Manager. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to Red Hat OpenShift Cluster Manager.
+
+.Procedure
+
+. Navigate to link:https://console.redhat.com/openshift/downloads[https://console.redhat.com/openshift/downloads]
+. From *Tokens* -> *Pull Secret*, click *Download*. 
++
+This downloads the file `pull-secret.txt` containing your `cloud.openshift.com` access token in JSON format:
++
+[source,json,subs="+quotes"]
+----
+{    
+  "auths": {        
+    "cloud.openshift.com": {            
+      "auth": "_<your_token>_",
+      "email": "_<email_address>_"
+    }
+}
+----
+
+. Download the global cluster pull secret to your local file system.
++
+[source,terminal]
+----
+$ oc extract secret/pull-secret -n openshift-config --to=.
+----
++
+[IMPORTANT]
+====
+Make a backup copy of your pull secret before updating. 
+====
+
+. In a text editor, edit the `.dockerconfigjson` file that was downloaded.
+. Copy the `"cloud.openshift.com"` JSON entry you downloaded into the `"auths"` entry in the file. Do not edit or delete any other entries. 
+. Save the file.
+
+You can now update your cluster to use this modified pull secret.
```

Comment:
Isn'ŧ it too general? Maybe not....There are more ways how you can do that:
1. Via UI by editing the pull-secret (not the yaml)
2. VIa UI by editing the yaml directly (you have to bear in mind that base64 encoding is involved)
3. Via oc

Reply:
We can do the UI. I tried to mirror the disable procedure, but I suppose we can update that as well.

Reply:
Okay I updated with what we discussed today.